README.md

```diff
@@ -1 +1,223 @@
+  parameters = {
+        bar = "example string"
+        foo = 123
+  }
+}
+```
+
+Then perform the following commands on the root folder:
+
+- `terraform init` to get the plugins
+- `terraform plan` to see the infrastructure plan
+- `terraform apply` to apply the infrastructure build
+- `terraform destroy` to destroy the built infrastructure
+
+
+[^]: (autogen_docs_start)
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| bucket\_region | The bucket's region location | string | `"us-central1"` | no |
+| job\_name | (Required) The name of the dataflow job | string | n/a | yes |
+| max\_workers | (Optional)  The number of workers permitted to work on the job. More workers may improve processing speed at additional cost. | string | `"1"` | no |
+| on\_delete | (Optional) One of drain or cancel. Specifies behavior of deletion during terraform destroy. The default is cancel. | string | `"cancel"` | no |
+| parameters | (Optional) Key/Value pairs to be passed to the Dataflow job (as used in the template). | map | `<map>` | no |
+| project\_id | (Required) The project in which the resource belongs. If it is not provided, the provider project is used. | string | n/a | yes |
+| service\_account\_email | (Optional) The Service Account email that will be used to identify the VMs in which the jobs are running | string | `""` | no |
+| temp\_gcs\_location | (Required) A writeable location on GCS for the Dataflow job to dump its temporary data. | string | n/a | yes |
+| template\_gcs\_path | (Required) The GCS path to the Dataflow job template. | string | n/a | yes |
+| zone | (Optional) The zone in which the created job should run. If it is not provided, the provider zone is used. | string | `"us-central1-a"` | no |
+
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| df\_job\_id | The unique Id of the newly created Dataflow job |
+| df\_job\_name | The name of the dataflow job |
+| df\_job\_state | The state of the newly created Dataflow job |
+| df\_job\_template\_gcs\_path | The GCS path to the Dataflow job template. |
+| temp\_gcs\_location | The GCS path for the Dataflow job's temporary data. |
+
+[^]: (autogen_docs_end)
+
+## Requirements
+
+Before this module can be used on a project, you must ensure that the following pre-requisites are fulfilled:
+
+1. Terraform is [installed](#software-dependencies) on the machine where Terraform is executed.
+2. The Service Account you execute the module with has the right [permissions](#configure-a-service-account).
+3. The necessary APIs are [active](#enable-apis) on the project.
+4. A working Dataflow template in uploaded in a GCS bucket
+
+The [project factory](https://github.com/terraform-google-modules/terraform-google-project-factory) can be used to provision projects with the correct APIs active.
+
+### Software Dependencies
+### Terraform
+- [Terraform](https://www.terraform.io/downloads.html) 0.10.x
```

Comment:
Our module template needs to be updated. :grimacing: 

```suggestion
- [Terraform](https://www.terraform.io/downloads.html) 0.11.x
```

Reply:
Resolved in
https://github.com/terraform-google-modules/terraform-google-dataflow/pull/1/commits/e5535bdba8de2d82c01c6d62f54b757cb706932d

Reply:
Resolved in
https://github.com/terraform-google-modules/terraform-google-dataflow/pull/1/commits/e5535bdba8de2d82c01c6d62f54b757cb706932d